Axolotl blog (#238)
brainglobe/brainglobe.github.io: docs/source/blog/axolotl-atlas-added.md

---
blogpost: true
date: August 09, 2024
author: Saima Abdus, Alessandro Felder, David Perez-Suarez
location: London, England
category: brainglobe
language: English
---

# An Atlas for the regenerative _Ambystoma mexicanum_ (axolotl) has been added to BrainGlobe

> Amphibians have long captivated human's interest. An unusual amphibian amongst these is the Axolotl (_Ambystoma mexicanum_), which is sometimes used as a model organism for regeneration. In 2021, [Lazcano et al.](https://www.nature.com/articles/s41598-021-89357-3), explored the axolotl's neuroanatomy via magnetic reasonance imaging (MRI) atlas of the juvenile axolotl brain. The axolotl atlas enables researchers to gain better insights into the processes underlying central nervous system (CNS) regeneration.

Reviewer: ```suggestion
Amphibians have long captivated human's interest. An unusual amphibian amongst these is the Axolotl (_Ambystoma mexicanum_), which is sometimes used as a model organism for regeneration. In 2021, [Lazcano et al.](https://www.nature.com/articles/s41598-021-89357-3) created a magnetic resonance imaging (MRI) atlas of the juvenile axolotl brain. The axolotl atlas enables researchers to gain better insights into the processes underlying central nervous system (CNS) regeneration.
```

![axolotl brain atlas annotations](./images/axolotl_annotations.png)

**Figure 1. Lateral view of the axolotl brain atlas annotations.**

> The Brainglobe Team re-packaged the data generated and made public by the original study, making it now possible to use the axolotl atlas within the BrainGlove ecosystem. The atlas name is `unam_axolotl_40um`.

Reviewer: ```suggestion
The BrainGlobe team re-packaged the data generated and made public by the original study, making it now possible to use the axolotl atlas within the BrainGlobe ecosystem. The atlas name is `unam_axolotl_40um`.
```

---
blogpost: true
date: August 09, 2024
author: Saima Abdus, Alessandro Felder, David Perez-Suarez
location: London, England
category: brainglobe
language: English
---

# An Atlas for the regenerative _Ambystoma mexicanum_ (axolotl) has been added to BrainGlobe

Amphibians have long captivated human's interest. An unusual amphibian amongst these is the Axolotl (_Ambystoma mexicanum_), which is sometimes used as a model organism for regeneration. In 2021, [Lazcano et al.](https://www.nature.com/articles/s41598-021-89357-3) created a magnetic resonance imaging (MRI) atlas of the juvenile axolotl brain. The axolotl atlas enables researchers to gain better insights into the processes underlying central nervous system (CNS) regeneration.

![axolotl brain atlas annotations](./images/axolotl_annotations.png)

**Figure 1. Lateral view of the axolotl brain atlas annotations and reference image.**

The BrainGlobe team re-packaged the data generated and made public by the original study, making it now possible to use the axolotl atlas within the BrainGlobe ecosystem. The atlas name is `unam_axolotl_40um`.

> Note that the reference image (as seen in Figure 1.) contains more than just the brain, so it will not be a good registration target for `brainreg`!

Reviewer: ```suggestion
Note that the reference image (as seen in Figure 1.) contains more than just the brain, so it may not be a good registration target for `brainreg`!
```

## How do I use the new atlas?

> You can use the axolotl atlas for visualisation (but not registration) like other BrainGlobe atlases. If you're interested in what a axolotl brain looks like, you could follow the steps below

Reviewer: ```suggestion
You can use the axolotl atlas for visualisation like other BrainGlobe atlases. If you're interested in what a axolotl brain looks like, you could follow the steps below
```
